hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
```

Comment:
oh you said "could", you mean "should" - as in "do it!"  ;-)  I see, you want me to subset the list.
Sorry. I misread what you meant.